lib/dokumi/tool/xcode/misplaced_constraint_finder.rb

```diff
@@ -0,0 +1,40 @@
+module Dokumi
+  module Tool
+    class Xcode
+      MisplacedStatus = Struct.new('MisplacedStatus', :file_path, :line_no)
```

Comment:
Most of the Dokumi codebase uses normal hashes, not structs for this type of code.